### ISSUE TITLE
Playback buttons don't work when using Spotify as AudioSource.

### ISSUE THREAD

**Issue by WillTheCoder10** [AudioBand/AudioBand#141]
## Description
I can't use any of the buttons when using the Spotify audio source, yet I see what song is playing and what the progress is.

## How To Reproduce
1. Go to Spotify, play any song.
2. Click on pause/play, skip song, or any similar buttons
3. Nothing happens. Song doesn't pause, shuffle/loop isn't enabled, song isn't skipped, etc.

## Version
 - Windows 11 Home; Version 23H2
 - AudioBand Version 1.2.4

## Logs/Screenshots Attachment
[AudioBand.log](https://github.com/user-attachments/files/16111961/AudioBand.log)
[Video showcasing issue](https://github.com/AudioBand/AudioBand/assets/87263327/1f3647d4-99a1-4928-806b-118d5ef9e09b)

**Comment by svr333** [comment]
An aware of the issue, haven't had a chance to look at it yet though.
The cause of the issue was Spotify changing some properties in their naming.

For context it was mentioned here in Discord: https://discord.com/channels/857720214619291658/870211206760255548/1256223860752257026

I will try my best to look at this issue somewhere in the coming week or so.

**Comment by MineTech99** [comment]
Same problem with me too, I hope you fix soon!

**Comment by ayyocrypto** [comment]
Same problem. Any updates on a fix?

**Comment by svr333** [comment]
I've tried looking at it but I'm unable to solve the issue, I'm not sure what's going wrong. If I do have some more time I will give it another look but for now the controls are limited to people with Premium.

Alternatively you can use the Windows 10 source with Spotify, though volume & liking will not work

**Comment by ayyocrypto** [comment]
Thanks for looking into it. I actually have premium though

**Comment by eamondaly** [comment]
Yep, I'm also a premium subscriber (premium family) and have been having the same issues. (Using the Windows 10 source works just fine, though.)

**Comment by svr333** [comment]
Hmm this should normally not cause any issues for Premium users, fixing that should be easier so I'll try to do that soon

**Comment by svr333** [comment]
Ah well so I found this related issue:
https://github.com/JohnnyCrazy/SpotifyAPI-NET/issues/980

Spotify changed the api to make it annoying to use, and the wrapper needs to update before it'll be fixed for Premium users.
Spotify also changed their app directly, meaning our workaround for free to use people is annoying to fix too (and I am not quite skilled enough to do Windows trickery)

For now I think we have to wait it out :/

**Comment by AnimaCaniba** [comment]
Happening to me as well:

Windows 10 Pro

I realize this is a 'wait and see' unfortunately due to Spotify, just wanted to include my version it is happening with in case the day comes there is a fix.

Thank you for a great product

**Comment by svr333** [comment]
Yup, I'll repeat for anyone seeing this, you can alternatively use the Windows 10 source in AudioBand, though volume & liking will not work when using it

**Comment by IZeeLikeEyeZee** [comment]
Hm, for me using the Windows 10 source just leaves it blank and then nothing does anything, is there anything I need to do to make it work?
